pkg/kev/converter/kubernetes/transform_test.go

```diff
@@ -70,58 +70,37 @@ var _ = Describe("Transform", func() {
 
 	Describe("Transform", func() {
 		When("service exclusion list is empty", func() {
-			Context("and has probe type", func() {
-				JustBeforeEach(func() {
-					projectService.Labels = composego.Labels{
-						config.LabelWorkloadLivenessProbeType: ProbeTypeNone.String(),
-					}
-
-					project.Services = []composego.ServiceConfig{
-						composego.ServiceConfig(projectService),
-					}
-
-					k = Kubernetes{
-						Opt:      ConvertOptions{},
-						Project:  &project,
-						Excluded: excluded,
-					}
-				})
-
-				It("includes kubernetes objects for project services", func() {
-					objs, err := k.Transform()
-					Expect(err).NotTo(HaveOccurred())
-					Expect(len(objs)).To(Equal(1))
+			BeforeEach(func() {
+				projectService.Labels = composego.Labels{
+					config.LabelWorkloadLivenessProbeType: ProbeTypeNone.String(),
+				}
+			})
 
-					u, err := ToUnstructured(objs[0])
-					name := u["metadata"].(map[string]interface{})["name"]
+			It("includes kubernetes objects for project services", func() {
+				objs, err := k.Transform()
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(objs)).To(Equal(1))
 
-					Expect(err).NotTo(HaveOccurred())
-					Expect(name).To(Equal(projectService.Name))
-				})
-			})
+				u, err := ToUnstructured(objs[0])
+				name := u["metadata"].(map[string]interface{})["name"]
 
-			Context("and no probe is defined", func() {
-				It("returns a missing probe type error", func() {
-					_, err := k.Transform()
-					Expect(err).To(HaveOccurred())
-					Expect(err.Error()).To(ContainSubstring("probe type not provided"))
-				})
+				Expect(err).NotTo(HaveOccurred())
+				Expect(name).To(Equal(projectService.Name))
 			})
-
 		})
 
-		When("excluded services are specified", func() {
+	})
 
-			BeforeEach(func() {
-				excluded = []string{projectService.Name}
-			})
+	When("excluded services are specified", func() {
```

Comment:
I guess this was moved out of the `Describe("Transform", func() {...` scope by mistake?